WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -51726,7 +51726,7 @@ b_theodosius = {
 	culture = romano_hispanic
 	religion = nicene
 }
-b_justinian = {
+b_Iustinian = {
```

Comment:
Since it is used elsewhere and isn't a localisation line, it is better to keep this as it is an change its localisation instead.

Reply:
I will fix it.